test/navigation/AuthenticatedNavigatorSpec.scala

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package navigation
+
+import base.SpecBase
+import models.{NormalMode, TaxYearSelection}
+import pages.authenticated.TaxYearSelectionPage
+import controllers.authenticated.routes._
+
+class AuthenticatedNavigatorSpec extends SpecBase {
+
+  val navigator = new AuthenticatedNavigator
+
+  "Authenticated Navigator" when {
+    "in Normal mode" must {
+
+      "from TaxYearSelection" must {
+
+        "go to TaxYearSelection when unanswered" in {
+          val answers = emptyUserAnswers
+
+          navigator.nextPage(TaxYearSelectionPage, NormalMode)(answers) mustBe
+            TaxYearSelectionController.onPageLoad(NormalMode)
+        }
+
+        "go to TaxYearSelection when answered" in {
+          val answers = emptyUserAnswers.set(TaxYearSelectionPage, Seq(TaxYearSelection.values.head)).success.value
```

Comment:
do we wanna do a gen here for TaxYearSelection values?

Reply:
Is that necessary as this page will always go to the same place, regardless of which box you check?